### PR TITLE
Add `final` and `@frozen`

### PR DESCRIPTION
**Pull request by ghost**
## Proposed changes

Add `final` and `@frozen`

## Types of changes

What types of changes does your code introduce to Frames Ios?
_Put an `x` in the boxes that apply_

* [ ] Bugfix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [ ] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: Source/CheckoutFont.swift [hunk 1]
```diff
@@ -13,7 +13,7 @@ extension UIFont {
 
 extension UIFont {
 
-    enum GraphikStyle: String, CaseIterable {
+    @frozen enum GraphikStyle: String, CaseIterable {
```
Comment (alex-ioja-yang-cko) [inline Source/CheckoutFont.swift:16]: It doesn't make sense to add @frozen to something that is not public

**Review by alex-ioja-yang-cko — CHANGES_REQUESTED** [review]
As mentioned in comment, @frozen should not be applied to something that's not public.

File: Source/CheckoutFont.swift (hunk 1 above)
Comment (ehab-al-cko) [reply Source/CheckoutFont.swift:16]: frozen : Apply this attribute to a structure or enumeration declaration to restrict the kinds of changes you can make to the type. This attribute is allowed only when compiling in library evolution mode. When the compiler isn’t in library evolution mode, all structures and enumerations are implicitly frozen, and this attribute is ignored.

**Review by alex-ioja-yang-cko — APPROVED** [review]
I'd remove the final from descriptions as I think its not providing much value, and I'd like to find a way and avoid introducing unintended white space changes, as they generally just cause noise and distract from the work being done.

Otherwise, absolutely making classes final should help to improve compile times to consumer.

**Review by deepesh-vasthimal-cko — APPROVED** [review]

File: Source/Core/InputState.swift [hunk 2]
```diff
@@ -5,7 +5,7 @@ import Foundation
 /// - normal
 /// - required
 /// - hidden
-public enum InputState {
+@frozen public  enum InputState {
```
Comment (harry-brown-cko) [inline Source/Core/InputState.swift:8]:
```suggestion
@frozen public enum InputState {
```

**Pull request merged**